deployment/nginx.conf.template

```diff
@@ -19,15 +19,11 @@ http {
     }
 
     location /auth/ {
-      proxy_pass ${CLIENT_AUTH_PROXY_URL};
-    }
-
-    location = /config {
-      alias ${PUBLIC_CONFIG_PATH};
+      proxy_pass ${CLIENT_TOKEN_EXCHANGE_PROXY_URL};
     }
 
     location / {
-      root /usr/share/nginx/html;
+      root /app/dist;
       try_files $uri /index.html;
     }
   }
```

Comment:
I think this file can also be removed, right?

Reply:
done